src/index.ts

```diff
@@ -249,6 +249,8 @@ export type FormConfig<T extends object> = {
    * [1]: https://github.com/jquense/yup
    */
   validationSchema?: Schema;
+
+  validationOption?: ValidateOptions;
```

Comment:
Please document this option and make it plural to follow type name.

```suggestion
  /**
   * [Yup][1] validation options used internally
   *
   * [1]: https://github.com/jquense/yup
   */
  validationOptions?: ValidateOptions;
```